docs/docs/guide/6_faq/index.md

```diff
@@ -11,4 +11,37 @@ The preferred solution is to use subdomains for each app. For example, you can u
 ## How can I use components that resemble the original components in the administration?
 While it is not possible to use the exact same components in the Shopware administration, there is a component library called Meteor Component Library that offers similar components. The Shopware administration components are not native Vue components because they have extension capabilities, Twig templates, and other features that cannot be directly used. However, by utilizing the Meteor Component Library, you can achieve a native look and feel for your app that seamlessly integrates with the original Shopware administration.
 
-To access the Meteor Component Library, visit the following link: https://github.com/shopware/meteor-component-library
\ No newline at end of file
+To access the Meteor Component Library, visit the following link: https://github.com/shopware/meteor-component-library
```

Comment:
We should redirect users to the new monorepo: https://github.com/shopware/meteor/tree/main/packages/component-library